src/views/v2/month.php

```diff
@@ -15,12 +15,24 @@
 
 $events = $this->get( 'events' );
 
-$this->template( 'events-bar' );
+?>
 
-$this->template( 'top-bar' );
+<div class="tribe-common-l-container tribe-events-l-container">
 
-?>
+	<?php $this->template( 'events-bar' ); ?>
+
+	<?php $this->template( 'top-bar' ); ?>
+
+	<div class="tribe-events-calendar-month">
```

Comment:
Are we missing the aria grid stuff here, or will this be getting merged with the work Juan has done on this view?